bin/main.rb

```diff
@@ -1,3 +1,92 @@
+# rubocop:disable Layout/LeadingCommentSpace, Lint/MissingCopEnableDirective, Metrics/MethodLength, Style/GlobalVars, Style/ClassVars, Style/MultipleComparison, Naming/VariableNumber, Layout/LineLength
 #!/usr/bin/env ruby
+class Board
+  private
 
-p 'hello world'
+  def line1
+    print ' ' * 6
+    print '#'
+    print ' ' * 5
+    print '#'
+    print "\n"
+  end
+
+  def line2
+    print ' '
+    print '# ' * 9
+    print "\n"
+  end
+
+  def true_line(array)
+    array = array.join('')
+    print array
+    print "\n"
+  end
+
+  public
+
+  def board
+    line1
+    true_line($arr[0])
+    line1
+    line2
+    line1
+    true_line($arr[1])
+    line1
+    line2
+    line1
+    true_line($arr[2])
+    line1
+  end
+
+  $arr = [[' ', ' ', ' ', ' ', ' ', ' ', '#', ' ', ' ', ' ', ' ', ' ', '#', ' ', ' ', ' ', ' ', ' '], [' ', ' ', ' ', ' ', ' ', ' ', '#', ' ', ' ', ' ', ' ', ' ', '#', ' ', ' ', ' ', ' ', ' '], [' ', ' ', ' ', ' ', ' ', ' ', '#', ' ', ' ', ' ', ' ', ' ', '#', ' ', ' ', ' ', ' ', ' ']]
+end
+
+class Game
+  @@turn = 1
+  def turns
+    p 'Firt put the a number between 1-3 to select the row then put a number between 1-3 to select the column'
+    while @@turn < 9
```

Comment:
- [ ] [REQUIRED] Usage of out-of-nowhere numbers to terminate a loop is not advised. Use a terminating variable instead. An example is shown below:
```
game_on = true

# initial instructions for player

while game_on
  # loop for each move
  
  if winner # and/or draw (the exact condition in this milestone is not important)
    game_on = false
  end
end

```